authenticators/jwt_from_extra_provider.go

```diff
@@ -0,0 +1,85 @@
+package authenticators
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/arrikto/oidc-authservice/common"
+	goidc "github.com/coreos/go-oidc"
+)
+
+const bearerPrefix = "Bearer "
+
+type JWTFromExtraProviderAuthenticator struct {
```

Comment:
We could probably unexport the struct, I don't users need to have access to this type